lmfit/parameter.py

```diff
@@ -587,6 +586,7 @@ def __init__(self, name=None, value=None, vary=True, min=-inf, max=inf,
         self.stderr = None
         self.correl = None
         self.from_internal = lambda val: val
+        self._val = value
```

Comment:
Maybe I am missing something, but this line is unchanged but just moved down?

Reply:
@Tillsten  Yes, that is true....